src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OrganisationsController.cs

```diff
@@ -30,12 +30,26 @@ public async Task<ActionResult> GetAllAsync()
 
             return Ok(new GetAllOrganisationsViewModel
             {
-                Organisations = organisationsList?.Select(x => new OrganisationViewModel
-                {
-                    OrganisationId = x.Id,
-                    Name = x.Name,
-                    OdsCode = x.OdsCode
-                })
+                Organisations = organisationsList?.Select(x =>
+                    new OrganisationViewModel
+                    {
+                        OrganisationId = x.Id,
+                        Name = x.Name,
+                        OdsCode = x.OdsCode,
+                        PrimaryRoleId = x.PrimaryRoleId,
+                        CatalogueAgreementSigned = x.CatalogueAgreementSigned,
+                        Location = x.Location == null ? null : new LocationViewModel
```

Comment:
The name of the Location property confused me slightly. It sounds more like a GPS location rather than a postal address. The agreed contract with the Frontend does method the property 'address', however it is in a different format. I spoke to @alibombaye & agreed the new JSON contract